.github/workflows/push.yml

```
@@ -68,7 +66,29 @@ jobs:
           push: true
           tags: ghcr.io/${{ github.repository }}:${{ needs.branch_meta.outputs.sha }}
           labels: ${{ steps.docker_meta_img.outputs.labels }}
-
+      - name: Docker meta Service Name (file storage)
+        id: docker_meta_img_file_storage
+        uses: crazy-max/ghaction-docker-meta@v4
```

the crazy-max github actions go to the docker github actions 
```suggestion
        uses: docker/ghaction-docker-meta@v4
```

---

docker/metadata-action@v4 ?